client-react/src/features/todos/store/actions/todoModals.actions.ts

```diff
@@ -2,12 +2,8 @@ import { createAction } from "@reduxjs/toolkit";
 import { Todo } from "../../models/todo.model";
```

Comment:
Keep naming convension the same accross the project. Use kebab-case

Reply:
fixed